catalog/transformer/catalog-transformer-streaming-impl/src/main/java/org/codice/ddf/transformer/xml/streaming/impl/Gml3ToWktImpl.java

```diff
@@ -13,43 +13,57 @@
  */
 package org.codice.ddf.transformer.xml.streaming.impl;
 
+import com.vividsolutions.jts.geom.Envelope;
 import com.vividsolutions.jts.geom.Geometry;
 import com.vividsolutions.jts.io.WKTWriter;
+import ddf.catalog.validation.ValidationException;
 import ddf.catalog.validation.impl.ValidationExceptionImpl;
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
 import java.util.Collections;
-import javax.xml.bind.JAXBElement;
-import net.opengis.gml.v_3_1_1.AbstractGeometryType;
-import org.codice.ddf.parser.Parser;
-import org.codice.ddf.parser.ParserConfigurator;
-import org.codice.ddf.parser.ParserException;
+import java.util.function.Function;
+import javax.xml.parsers.ParserConfigurationException;
 import org.codice.ddf.transformer.xml.streaming.Gml3ToWkt;
-import org.jvnet.jaxb2_commons.locator.DefaultRootObjectLocator;
-import org.jvnet.ogc.gml.v_3_1_1.jts.ConversionFailedException;
-import org.jvnet.ogc.gml.v_3_1_1.jts.GML311ToJTSGeometryConverter;
+import org.geotools.geometry.jts.JTS;
+import org.geotools.referencing.CRS;
+import org.geotools.referencing.crs.DefaultGeographicCRS;
+import org.geotools.xml.Configuration;
+import org.opengis.referencing.FactoryException;
+import org.opengis.referencing.operation.MathTransform;
+import org.opengis.referencing.operation.TransformException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.xml.sax.SAXException;
 
 public class Gml3ToWktImpl implements Gml3ToWkt {
-  private final Parser parser;
+  private static final Logger LOGGER = LoggerFactory.getLogger(Gml3ToWkt.class);
 
-  private final ParserConfigurator configurator;
+  private static final String EPSG_4326 = "EPSG:4326";
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(Gml3ToWkt.class);
+  private final MathTransform latLonTransform;
+
+  private final org.geotools.xml.Parser parser;
 
-  public Gml3ToWktImpl(Parser parser) {
-    this.parser = parser;
-    this.configurator =
-        parser.configureParser(
-            Collections.singletonList(AbstractGeometryType.class.getPackage().getName()),
-            Gml3ToWktImpl.class.getClassLoader());
+  private final WKTWriter wktWriter = new WKTWriter();
```

Comment:
To my knowledge, the WKTWriter isn't thread-safe and as a result we have been wrapping it in ThreadLocal
https://github.com/codice/ddf/blob/master/catalog/spatial/kml/spatial-kml-transformer/src/main/java/org/codice/ddf/spatial/kml/converter/KmlToMetacard.java#L60

Reply:
Wrapped in a thread local like the example, thanks!